.circleci/config.yml

```diff
@@ -35,6 +35,16 @@ jobs:
           paths:
             - ./node_modules
 
+      - deploy:
+          name: deployment
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "develop" ]; then
+              cat deploy.sh | ssh -o "StrictHostKeyChecking no" $SSH_USER_DEVELOP@$SSH_HOST_DEVELOP CIRCLE_BRANCH=develop 'bash -'
```

Comment:
Where are the variables `SSH_USER_DEVELOP` and `SSH_HOST_DEVELOP` set?

Reply:
You can set up environment variables at the CI level. I put those in environment variables to avoid exposing the ip of our servers in source code in case there's some bot crawling ips to attack.

Reply:
Makes sense. Thanks!